audb/core/load.py

```diff
@@ -387,6 +554,15 @@ def _remove_media(
         )
 
 
+def _replace_ext(
+        file: str,
+        new_ext: str,
+) -> str:
+    file, _ = os.path.splitext(file)
+    file = f'{file}.{new_ext}'
+    return file
```

Comment:
Should we maybe add `audeer.new_file_extension()`?

Reply:
Yes, I was actually looking for such a function in `audeer` before I implemented it :)

Reply:
I created https://github.com/audeering/audeer/pull/40

Reply:
cool, let me know when you made a new release that I can use the function here

Reply:
It's now available with `audb` 1.14.0.

Reply:
I guess you mean `audeer` 1.14.0 :)

Reply:
done